changelog/1236.feature.rst

```diff
@@ -0,0 +1,3 @@
+Added a function plasmapy.thomson.Maxwellian_scattered_power which computes the Thomson-scattered power spectrum (to a constant factor) to first order in beta.
+Renamed plasmapy.thomson.spectral_density to plasmapy.thomson.Maxwellian_spectral_density.
```

Comment:
Format these like \`~plasmapy.thomson.spectral_density\` to create links in the changelog back to the functions